File: diagnostics-web/src/app/Model/DiagnosticMsg.ts
import {SystemEvent} from './DiagResponse';
import {IFilterableEvent} from './IFilterableEvent';

export class DiagnosticMsg extends SystemEvent implements IFilterableEvent {
  level = 0;
  id = 0;
  machine = '';
  message = '';
  process = '';
  user = '';
  isSelected = false;
  msgId = '';
}

File: DiagnosticExplorer/DiagnosticMsg.cs
using System;

namespace DiagnosticExplorer;
public class DiagnosticMsg
{
	public int Level { get; set; }
	public DateTime Date { get; set; }
	public string Machine { get; set; }
	public string Process { get; set; }
	public string User { get; set; }
	public string Category { get; set; }
	public string Message { get; set; }
	public string Environment{ get; set; }

}